include/z64.h

```diff
@@ -223,14 +223,14 @@ typedef struct {
 
 typedef struct {
     /* 0x00 */ u8*      texture;
-    /* 0x04 */ s16      unk_4;
-    /* 0x06 */ s16      unk_6;
-    /* 0x08 */ u8       unk_8;
-    /* 0x09 */ u8       unk_9;
-    /* 0x0A */ u8       delayA;
-    /* 0x0B */ u8       delayB;
-    /* 0x0C */ s16      unk_C;
-    /* 0x0E */ s16      unk_E;
+    /* 0x04 */ s16      x;
+    /* 0x06 */ s16      y;
+    /* 0x08 */ u8       width;
+    /* 0x09 */ u8       height;
+    /* 0x0A */ u8       timer;
+    /* 0x0B */ u8       delay;
```

Comment:
mostly ok with this name, just wondering if it might be worth distinguishing between `delay` and `timer`. I assume `delay` is how long it takes to start appearing, and `timer` is how long its visible for.
maybe `duration` is better than timer? or just keep that names and put a comment next to them. 
others should weigh in, cause im not too sure

Reply:
duration actually probably implies the whole duration, and not whats left. but yea maybe just some clarifying comments since theyre both technically timers

Reply:
Yeah I don't know about naming it `duration` since it gets decremented over time. Maybe something like `remainingTime` would be better. And I think even `timer` would be less confusing than `duration` tbh.

Reply:
Alternatively, we could go with a naming pattern like `delayTimer` and `durationTimer`, to make it clear that they both decrement over time. And in the function where they get set, we can leave the arguments as `delay`/`duration` since that's still correct at that point.

Reply:
yea im cool with `delayTimer` and `durationTimer`